ui/src/assets/i18n/en.json

```diff
@@ -88,6 +88,7 @@
   "application_repoman_cant_link": "here",
   "application_repo_title": "Link application to a repository: ",
   "application_repo_strat_title" : "Configuration",
+  "application_deployment_title" : "Link application to a deployment platform: ",
```

Comment:
key deployment_platform_name is missing (used in application -> advanced)

Reply:
done